tests/pbft_manager_test.cpp

```diff
@@ -46,24 +46,24 @@ void check_2tPlus1_validVotingPlayers_activePlayers_threshold(size_t committee_s
   auto nonce = 1;  // fixme: the following nonce approach is not correct anyway
   uint64_t trxs_count = 0;
 
-  {
-    auto min_stake_to_vote = node_cfgs[0].chain.final_chain.state.dpos->eligibility_balance_threshold;
-    state_api::DPOSTransfers delegations;
-    for (size_t i(1); i < nodes.size(); ++i) {
-      std::cout << "Delegating stake of " << min_stake_to_vote << " to node " << i << std::endl;
-      node_1_expected_bal -= delegations[nodes[i]->getAddress()].value = min_stake_to_vote;
-    }
-    auto trx = make_dpos_trx(node_cfgs[0], delegations, nonce++);
-    nodes[0]->getTransactionManager()->insertTransaction(trx);
-    trxs_count++;
-    EXPECT_HAPPENS({120s, 1s}, [&](auto &ctx) {
-      for (auto &node : nodes) {
-        if (ctx.fail_if(!node->getFinalChain()->transaction_location(trx->getHash()))) {
-          return;
-        }
-      }
-    });
-  }
+  // {
+  //   auto min_stake_to_vote = node_cfgs[0].chain.final_chain.state.dpos->eligibility_balance_threshold;
+  //   state_api::DPOSTransfers delegations;
+  //   for (size_t i(1); i < nodes.size(); ++i) {
+  //     std::cout << "Delegating stake of " << min_stake_to_vote << " to node " << i << std::endl;
+  //     node_1_expected_bal -= delegations[nodes[i]->getAddress()].value = min_stake_to_vote;
+  //   }
+  //   auto trx = make_dpos_trx(node_cfgs[0], delegations, nonce++);
+  //   nodes[0]->getTransactionManager()->insertTransaction(trx);
+  //   trxs_count++;
+  //   EXPECT_HAPPENS({120s, 1s}, [&](auto &ctx) {
+  //     for (auto &node : nodes) {
+  //       if (ctx.fail_if(!node->getFinalChain()->transaction_location(trx->getHash()))) {
+  //         return;
+  //       }
+  //     }
+  //   });
+  // }
```

Comment:
The part of code is for delegate different amount of stake to nodes. I don't understand why comment out

Reply:
we do not have same kind of transactions to delegate, it's commented out as this needs to be reworked